.github/run_erblint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+shopt -s globstar
+
+bundle exec erblint --lint-all
+
+# Store the return code of the erblint execution
+EXIT_CODE=$?
+
+shopt -u globstar
+
+exit $EXIT_CODE
```

Comment:
Please check permissions on file

Reply:
Done : just needed to `git add chmod=+x run_erblint.sh`